.github/workflows/publish.yaml

```diff
@@ -0,0 +1,32 @@
+name: Publish
+
+on:
+  pull_request:
+    types: [closed]
```

Comment:
Is it means that version should be created even if the PR vas closed and not merged?

Reply:
We have conditional job execution:
```
if: github.event.pull_request.merged && github.head_ref == 'version-bump'
```

Reply:
Good, thnx!